extension-localization/src/main/java/com/mapbox/maps/extension/localization/Localization.kt

```diff
@@ -25,6 +33,21 @@ internal fun setMapLanguage(locale: Locale, style: StyleInterface, layerIds: Lis
     return
   }
 
+  /**
+   * Check if the server side localization is set and Log the error if present.
+   * `localizeLabel` method (ie Style localization) will not work with server side localization
+   */
+  checkServerSideLocalizationSet { isLocalizationSet ->
+    if (isLocalizationSet) {
+      Logger.e(
+        TAG,
+        "Style localization will not work when language or worldview is set." +
+          " Either remove localizeLabel method or remove server side localization " +
```

Comment:
🤔 how method can be removed? `Either remove localizeLabel method`

Reply:
will fix the error language, this log is to let user know that client-side localization won't work with server-side implementation.

Reply:
wondering if we could replace this logic with a lint setup instead.. might be a bit too involved..